dist/lib/components/box/box.d.ts

```diff
@@ -2,7 +2,7 @@
 import { BackgroundProps, BorderProps, ColorProps, FlexboxProps, GridProps, LayoutProps, PositionProps, ShadowProps, SpaceProps } from 'styled-system';
 interface Props extends BackgroundProps, BorderProps, ShadowProps, SpaceProps, ColorProps, FlexboxProps, GridProps, LayoutProps, PositionProps {
     color?: string;
-    as?: 'div' | 'section' | 'main' | 'aside' | 'article';
+    as?: 'div' | 'section' | 'main' | 'aside' | 'article' | 'header';
```

Comment:
@opaquejacob let's add footer as well while we are at it.

Reply:
Done